js_ts/nodejs-ts/.gitignore

```diff
@@ -0,0 +1,2 @@
+package-lock.json
```

Comment:
JS comments also apply to the TS side. I'll leave adding these to the TS side to you @turalsalamov